esphome/components/pmsa003i/pmsa003i.cpp

```diff
@@ -7,14 +7,23 @@ namespace pmsa003i {
 
 static const char *const TAG = "pmsa003i";
 
+static const uint8_t START_CHARACTER_1 = 0x42;
+static const uint8_t START_CHARACTER_2 = 0x4D;
+static const uint8_t COUNT_START_CHARACTER_BYTES = 2;
+static const uint8_t COUNT_PAYLOAD_LENGTH_BYTES = 2;
+static const uint8_t COUNT_PAYLOAD_BYTES = 28;
+static const uint8_t COUNT_16_BIT_VALUES = (COUNT_PAYLOAD_LENGTH_BYTES + COUNT_PAYLOAD_BYTES) / 2;
+static const uint8_t COUNT_DATA_BYTES = COUNT_START_CHARACTER_BYTES + COUNT_PAYLOAD_LENGTH_BYTES + COUNT_PAYLOAD_BYTES;
+static const uint8_t CHECKSUM_START_INDEX = COUNT_DATA_BYTES - 2;
```

Comment:
Try to keep constants at least somewhat alphabetized.
```suggestion
static const uint8_t COUNT_PAYLOAD_BYTES = 28;
static const uint8_t COUNT_PAYLOAD_LENGTH_BYTES = 2;
static const uint8_t COUNT_START_CHARACTER_BYTES = 2;
static const uint8_t COUNT_DATA_BYTES = COUNT_START_CHARACTER_BYTES + COUNT_PAYLOAD_LENGTH_BYTES + COUNT_PAYLOAD_BYTES;
static const uint8_t CHECKSUM_START_INDEX = COUNT_DATA_BYTES - 2;
static const uint8_t COUNT_16_BIT_VALUES = (COUNT_PAYLOAD_LENGTH_BYTES + COUNT_PAYLOAD_BYTES) / 2;
static const uint8_t START_CHARACTER_1 = 0x42;
static const uint8_t START_CHARACTER_2 = 0x4D;
```